### ISSUE TITLE
Error building for android or iOS

### ISSUE THREAD

**Issue by mugikhan** [Sh1d0w/multi_image_picker#137]
**Bug Report**

`multi_image_picker` version: 4.2.0
Is it happening on Android: Yes

Is it happening on iOS: Yes

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.5.9-pre.111, on Mac OS X 10.14.4 18E226, locale en-ZA)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
[!] Android Studio (version 3.2)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] IntelliJ IDEA Ultimate Edition (version 2019.1.1)
[!] VS Code (version 1.33.1)
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
[✓] Connected device (1 available)

! Doctor found issues in 2 categories.
```

**Describe the bug**

Error when building for android or iOS

```
Compiler message:
file:///Users/mugheeskhan/flutter/.pub-cache/hosted/pub.dartlang.org/multi_image_picker-4.2.0/lib/src/asset_thumb_provider.dart:35:31: Error: The argument type 'Null Function(StringBuffer)' can't be assigned to the parameter type 'Iterable<DiagnosticsNode> Function()'.
 - 'StringBuffer' is from 'dart:core'.
 - 'Iterable' is from 'dart:core'.
 - 'DiagnosticsNode' is from 'package:flutter/src/foundation/diagnostics.dart' ('file:///Users/mugheeskhan/flutter/packages/flutter/lib/src/foundation/diagnostics.dart').
Try changing the type of the parameter, or casting the argument to 'Iterable<DiagnosticsNode> Function()'.
        informationCollector: (StringBuffer information) {
                              ^
```

**Comment by Sh1d0w** [comment]
This plugin supports only Flutter in the Stable channel, which is version 1.2.1

You are using version 1.5.9 which is unstable. I would suggest you to never build against unstable versions and use the stable one.

Closing as duplicate of https://github.com/Sh1d0w/multi_image_picker/issues/135

**Comment by Sh1d0w** [comment]
I've released version 4.2.1 which should solve the problem.

Again, it is good practice to use only stable version of Flutter.
